build.gradle.kts

```diff
@@ -275,6 +275,21 @@ if (target == ProjectTarget.MAIN) {
     }
 }
 
+fun includeBuildPaths(buildPathsFile: File, sourceSet: Provider<SourceSet>) {
+    if (buildPathsFile.exists()) {
+        sourceSet.get().apply {
+            val buildPaths = buildPathsFile.readText().lineSequence()
+                .map { it.substringBefore("#").trim() }
+                .filter { it.isNotBlank() }
+                .toSet()
+            kotlin.include(buildPaths)
+            java.include(buildPaths)
+        }
+    }
+}
+includeBuildPaths(file("buildpaths.txt"), sourceSets.main)
+includeBuildPaths(file("buildpaths-test.txt"), sourceSets.test)
```

Comment:
I'm probably doing something wrong but this causes me to get the following error when building with
`skyhanni.multi-version=compile` in my .gradle/private.properties
I've tried with both java 8 and java 21